Jenkinsfile.release

```diff
@@ -0,0 +1,175 @@
+    ])
+  },
+  testResults: ['**/validate-expectations-results.xml'],
+  vulnerabilityScan: {
+    nexusPolicyEvaluation(
+      iqApplication: 'docker-nexus-iq-server',
+      iqScanPatterns: [[scanPattern: "container:${env.DOCKER_IMAGE_ID}"]],
+      iqStage: 'release')
+  },
+  deploy: {
+    pushImage()
+  },
+  postDeploy: {
+    sshagent(credentials: [sonatypeZionCredentialsId()]) {
+      sh '''git config user.email "sonatype-zion@sonatype.com"
+            git tag ${env.VERSION}
+            git push origin ${env.VERSION}'''
+    }
+  },
+  onSuccess: {
+    githubStatusUpdate('success')
+  },
+  onFailure: {
+    githubStatusUpdate('failure')
+  }
+)
+
+void updateIQServerVersionAndChecksum(String version, String checksum) {
+  def dockerFile = readFile(file: "${pwd()}/Dockerfile")
+  def versionRegex = /(ARG IQ_SERVER_VERSION=)(\d\.\d{1,3}\.\d\-\d{2})/
+  def shaRegex = /(ARG IQ_SERVER_SHA256=)([A-Fa-f0-9]{64})/
+
+  dockerFile = dockerFile.replaceAll(versionRegex, "\$1${version}")
+  dockerFile = dockerFile.replaceAll(shaRegex, "\$1${checksum}")
+
+  writeFile(file: "${pwd()}/Dockerfile", text: dockerFile)
+}
+
+void commitAndPushChanges(String version) {
+  runSafely 'git config --global push.default simple'
+  sonatypeZionGitConfig()
+  sshagent(credentials: [sonatypeZionCredentialsId()]) {
+    runSafely 'git add .'
+    runSafely "git diff --exit-code --cached || git commit -m 'Update to version ${version}'"
+
+    // pull and merge any new commits on main so that the push doesn't fail
+   runSafely 'git pull --no-rebase --no-edit origin main'
+   runSafely 'git push origin HEAD:main'
+  }
+}
+
+void pushImage() {
+  runSafely "mkdir -p '${env.WORKSPACE_TMP}/.dockerConfig'"
+  runSafely "cp -n '${env.HOME}/.docker/config.json' '${env.WORKSPACE_TMP}/.dockerConfig' || true"
+
+  withEnv(["DOCKER_CONFIG=${env.WORKSPACE_TMP}/.dockerConfig", 'DOCKER_CONTENT_TRUST=1']) {
+    withCredentials([
+      file(credentialsId: 'nexus-iq-server-repository-key', variable: 'NEXUS_IQ_SERVER_REPOSITORY_KEY'),
+      file(credentialsId: 'sonatype-pub', variable: 'SONATYPE_PUB'),
+      file(credentialsId: 'sonatype-key', variable: 'SONATYPE_KEY'),
+      [ $class: 'UsernamePasswordMultiBinding', credentialsId: 'docker-hub-credentials',
+        usernameVariable: 'DOCKERHUB_API_USERNAME', passwordVariable: 'DOCKERHUB_API_PASSWORD']]) {
+
+      runSafely """docker login --username ${env.DOCKERHUB_API_USERNAME} --password ${env.DOCKERHUB_API_PASSWORD}"
+                   docker trust key load $NEXUS_IQ_SERVER_REPOSITORY_KEY
+                   docker trust key load $SONATYPE_KEY"""
+
+      // add signer - for this you need signers public key and repository keys password
+      withCredentials([string(credentialsId: 'nexus-iq-server_dct_reg_pw', variable: 'DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE')]) {
+        runSafely "docker trust signer add sonatype ${organization}/${dockerHubRepository} --key $SONATYPE_PUB"
```

Comment:
note that your $org/$repo here is already declared as the `imageName` variable above, can probably clean that up a bit

Reply:
Addressed: https://github.com/sonatype/docker-nexus-iq-server/blob/CLM-22746_update_builder_node/Jenkinsfile.release#L138